First_GUI.py

```diff
@@ -1,16 +1,17 @@
-from Tkinter import *
-import tkMessageBox
-import Tkinter
+import tkinter as tk
```

Comment:
tkinter is all lower case in Python3. Also, instead of importing the individual modules within tkinter you can just access them with (tk.thenameofthething)